pages/agent/v3/windows.md.erb

```diff
@@ -4,42 +4,36 @@ The Buildkite Agent is supported on Windows 8, Windows Server 2012, and newer.
 
 {:toc}
 
-## Automated install with Powershell
+## Automated install with PowerShell
 
-You'll need to run the automated installer within Powershell with Administrative Privileges.
+You'll need to run the automated installer within PowerShell with administrative privileges.
 
-To start a Powershell session as an Administrator:
+Once you're in an escalated PowerShell session, you can run this script to install the latest version of the agent:
 
-1. Open the Start menu and type "powershell"
-2. Highlight "Windows Powershell" in the results
-3. Press CTRL+SHIFT+ENTER (you'll be prompted to allow this app to make changes)
-
-Once you're in an escalated Powershell session, you can run this script to install the latest version of the Agent:
-
-```
-$env:buildkiteAgentToken = "my_token"
-Set-ExecutionPolicy Bypass -Scope Process -Force
+```shell
+PS> $env:buildkiteAgentToken = "<your_token>"
+PS> Set-ExecutionPolicy Bypass -Scope Process -Force
 iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/buildkite/agent/main/install.ps1'))
 ```
 
-## Manual Installation
+## Manual installation
```

Comment:
Do you need admin privileges to run this? Do we prefer one over the other? At the mo it isn't super clear that you don't need to run both. Maybe a short line to the effect of

> If you can't install using the autaomtic method you can use this one?

Reply:
Manual installation doesn't run anything, you're literally just copying files around. They might need admin privileges depending on where they decide to copy them to/any restrictions on their computer, but that really is into "can they manage their own machine" territory. I've added a sentence up top to make clear there are two installation methods.